src/common/components/Navbar.vue

```diff
@@ -265,13 +266,15 @@ export default {
     },
 
 
-    handleNotificationRead(notif) {
+    async handleNotificationRead(notif) {
+      clearTimeout(timeout)
+      if (notif.read) return
+
       notif.read = true
-      this.$store.dispatch("substrate/updateDataListNotification", {
-        address: this.wallet.address,
-        role: "customer",
-        data: this.notifications
-      })
+      this.notifReads.push(notif.id)
+      timeout = setTimeout(async () => {
+        await setReadNotification(this.notifReads)
+      }, 2000)
```

Comment:
Why is there a timeout?

Reply:
to debounce and reduce bulk request to API

Reply:
I don't think this is how debounces work, but I will approve this one for now with  a note to change later.